app/src/main/java/org/mozilla/tv/firefox/pocket/PocketEndpoint.kt

```diff
@@ -61,9 +62,10 @@ private object PocketEndpointRaw {
 
     /** @return The global video recommendations as a raw JSON str or null on error. */
     @AnyThread // executeAndAwait hands off the request to the OkHttp dispatcher.
-    suspend fun getGlobalVideoRecommendations(): String? {
+    suspend fun getGlobalVideoRecommendations(version: String): String? {
         val req = Request.Builder()
                 .url(GLOBAL_VIDEO_ENDPOINT.toString())
+                .header("User-Agent", "Firefox TV $version")
```

Comment:
Nit: this will probably be easier for them to parse without spaces.  Can we replace them with dashes?